### PR TITLE
[windows] Modify pack script to enable webapi test suite msi package

### PR DESCRIPTION
**Pull request by zhuyongyong**
- It's fine to pack msi package with the template manifest.json file

https://crosswalk-project.org/jira/browse/XWALK-5632

### CODE REVIEW

File: tools/build/pack_windows.py [hunk 1]
```diff
@@ -504,18 +508,28 @@ def buildSubAPP(app_dir=None, build_json=None, app_dest_default=None):
...
+        if not doCopy(os.path.join(BUILD_ROOT_SRC, "icon.png"),
+                    os.path.join(BUILD_ROOT_SRC_PKG_APP, "icon.ico")):
+            return False
 
     hosted_app = False
     if safelyGetValue(build_json, "hosted-app") == "true":
         hosted_app = True
+
+    if not os.path.exists(BUILD_ROOT_SRC_PKG_APP):
+		doCMD("mkdir %s" % BUILD_ROOT_SRC_PKG_APP, DEFAULT_CMD_TIMEOUT)
```
Comment (haoyunfeix) [inline tools/build/pack_windows.py]: On windows OS, we'd better use "os.mkdirs" instead of "mkdir"

File: tools/build/pack_windows.py [hunk 2]
```diff
@@ -392,35 +392,39 @@ def packMsi(build_json=None, app_src=None, app_dest=None, app_name=None):
...
-    if not doCMD(create_cmd, DEFAULT_CMD_TIMEOUT):
-        os.chdir(orig_dir)
-        return False
+    if not safelyGetValue(build_json, "apk-type") or safelyGetValue(build_json, "apk-type") != "MANIFEST":
+        if os.path.exists(os.path.join(app_src, "manifest.json")):
+            if not doRemove([os.path.join(app_src, "manifest.json")]):
+                os.chdir(orig_dir)
+                return False
+        if os.path.exists(os.path.join(app_src, "icon.ico")):
+            build_cmd = "node %s/src/crosswalk-pkg -c crosswalk-%s.zip --platforms=windows -m " \
```
Comment (haoyunfeix) [inline tools/build/pack_windows.py]: -c parameter, we could use CROSSWALK_VERSION to download crosswalk-*.zip. by itself

Comment (zhuyongyong) [inline tools/build/pack_windows.py]: The apptools for windows can't download crosswalk-*.zip. by itself

Comment (Honry) [inline tools/build/pack_windows.py]: @zhuyongyong, apptools on Windows support automatically download Crosswalk Canary version currently.

**Comment by jennycao** [comment]
merge this PR first before branch crosswalk 17, and create a seperate PR task for autodownload crosswalk during build.

**Pull request merged**